Cargo.toml

```
@@ -43,3 +43,8 @@ windows = { version = "0.51.1", features = ["Win32_UI_Shell", "Win32_Foundation"
 
 [patch.crates-io]
 async_zip = { git = "https://github.com/Majored/rs-async-zip.git", rev = "a58024e" }
+
+[features]
+default = ["native-tls"]
```

特に理由がないので各種クロスビルドが通るならデフォルトrustlsに変えていい気もしてます

---

(切り替えると #231 も実現されそう)

---

`aarch64-apple-darwin`だと[通らなさそう](https://github.com/briansmith/ring/issues/1442)なので互換性のためにそのままにしてあります。

---

Darwinビルドはmacos sdkのライセンスの都合でクロスコンパイルできないので通ると思うので変えちゃってください

一応ローカルのmacで確認しときます

---

ビルド通ったのでデフォルトrustlsにします
```suggestion
default = ["rustls"]
```